apps/web/pages/v2/availability/[schedule].tsx

```diff
@@ -132,7 +134,17 @@ export default function Availability({ schedule }: { schedule: number }) {
                 <h3 className="mb-5 text-base font-medium leading-6 text-gray-900">
                   {t("change_start_end")}
                 </h3>
-                <Schedule control={control} name="schedule" />
+                {typeof me.data?.weekStart === "string" && (
+                  <Schedule
+                    control={control}
+                    name="schedule"
+                    weekStart={
+                      ["Sunday", "Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday"].indexOf(
+                        me.data?.weekStart
+                      ) as 0 | 1 | 2 | 3 | 4 | 5 | 6
```

Comment:
```suggestion
                      )
```

Reply:
weekStart prop is a `number` type and thus it doesn't need type assertion
Also, the result can be -1 as well so assertion isn't correct.

Reply:
This is fair, it's a copy paste from similar places (I'm debating with myself whether to move weekStart logic into the schedule component itself).

Reply:
maybe we can have MySchedule Higher Order Component that can have user specific settings preconfigured.